app/controllers/likes_controller.rb

```diff
@@ -1,7 +1,8 @@
 class LikesController < ApplicationController
   def create
     @post = Post.find(params[:post_id])
-    @like = @post.likes.build(user_id: params[:user_id])
+    @like = @post.likes.new(user_id: params[:user_id]) # Use `new` instead of `build`
```

Comment:
- [ ] [REQUIRED] I noticed that some tests are still failed :cry: . Please make sure that you fix all the failed tests below :point_down:  before submitting your project for another review. It is very important to make all your tests passed as required for this project :+1: 

![failed](https://github.com/Odongo006/Rails_blog_App/assets/6615394/6601a37c-a9bb-42df-92af-c1161c245c0e)